view/frontend/web/js/view/payment/method-renderer/omise-offline-tesco-method.js

```diff
@@ -0,0 +1,140 @@
+define(
+    [
+        'jquery',
+        'ko',
+        'mage/storage',
+        'Magento_Checkout/js/view/payment/default',
+        'Magento_Checkout/js/model/full-screen-loader',
+        'Magento_Checkout/js/action/redirect-on-success',
+        'Magento_Checkout/js/model/quote',
+        'Magento_Checkout/js/model/url-builder',
+        'Magento_Checkout/js/model/error-processor'
+    ],
+    function (
+        $,
+        ko,
+        storage,
+        Component,
+        fullScreenLoader,
+        redirectOnSuccessAction,
+        quote,
+        urlBuilder,
+        errorProcessor
+    ) {
+        'use strict';
+
+        return Component.extend({
+            defaults: {
+                template: 'Omise_Payment/payment/offline-tesco-form'
+            },
+
+            isPlaceOrderActionAllowed: ko.observable(quote.billingAddress() != null),
+
+            /**
+             * Get payment method code
+             *
+             * @return {string}
+             */
+            getCode: function() {
+                return 'omise_offline_tesco';
+            },
+
+            /**
+             * Initiate observable fields
+             *
+             * @return this
+             */
+            initObservable: function() {
+                this._super()
+                    .observe([
```

Comment:
@jacekstan do we need to observe anything here? Seems this part is unnecessary since there is no additional information that needs to be submitted from the checkout page

Reply:
well.. we do not..